### PR TITLE
This removes all the comments about fission

### PR DESCRIPTION
**Pull request by mischat**

### CODE REVIEW

File: contracts/mocks/MockBosonRouter.sol [hunk 1]
```diff
@@ -149,11 +149,11 @@ contract MockBosonRouter is
...
-        notAboveETHLimit(_metadata[2].mul(_metadata[5]));
-        notAboveETHLimit(_metadata[3].mul(_metadata[5]));
-        notAboveETHLimit(_metadata[4].mul(_metadata[5]));
-        require(_metadata[3].mul(_metadata[5]) == msg.value, "IF"); //invalid funds
-        //hex"54" FISSION.code(FISSION.Category.Finance, FISSION.Status.InsufficientFunds)
+        notAboveETHLimit(metadata[2].mul(metadata[5]));
+        notAboveETHLimit(metadata[3].mul(metadata[5]));
+        notAboveETHLimit(metadata[4].mul(metadata[5]));
+        require(metadata[3].mul(metadata[5]) == msg.value, "IF"); //invalid funds
+
```
Comment (zajck) [inline contracts/mocks/MockBosonRouter.sol:157]: Some automatic merge probably went wrong here.
`_metadata` should be correct.
That's also why build is failing.

Same is true for next three changes in this file.

**Review by zajck — CHANGES_REQUESTED** [review]

File: contracts/mocks/MockBosonRouter.sol (hunk 1 above)
Comment (hswopeams) [reply contracts/mocks/MockBosonRouter.sol]: I merged with main. This issue should be resolved.

**Review by hswopeams — DISMISSED** [review]

**Review by zajck — DISMISSED** [review]

File: contracts/mocks/MockBosonRouter.sol [hunk 2]
```diff
@@ -169,7 +169,7 @@ contract MockBosonRouter is
         whenNotPaused
     {
         checkLimits(_metadata, address(0), address(0), 0);
-        requestCreateOrder(_metadata, 5, address(0), address(0), 0);
+        requestCreateOrder(_metadata, ETHETH, address(0), address(0), 0);
```
Comment (bidyut-arianelabs) [inline contracts/mocks/MockBosonRouter.sol:172]: Is this intentional?

Comment (hswopeams) [reply contracts/mocks/MockBosonRouter.sol:172]: This is intentional. The` 5` has to be in there in order to make one of the test cases fail. It had to be added back in  anytime BosonRouter.sol changes are copied into MockBosonRouter.sol. There were too many changes in BosonRouter to change them one by one in MockBosonRouter.sol and expect to get them right.

**Review by bidyut-arianelabs — APPROVED** [review]
Looks good to me 👍

**Review by zajck — APPROVED** [review]

**Pull request merged**